### ISSUE TITLE
training epoch scheduling

### ISSUE THREAD

**Issue by shanggdlk** [open-mmlab/mmdetection#3031]
Hi, there,

I saw on https://mmdetection.readthedocs.io/en/latest/tutorials/finetune.html training epoch settings, there is a comment saying **actual epoch = 8 * 8 = 64** (8 is the batch size). Does that mean we iterate the entire dataset for 64 times?

I highlight the comments in bold below.

# optimizer
# lr is set for a batch size of 8
optimizer = dict(type='SGD', lr=0.01, momentum=0.9, weight_decay=0.0001)
optimizer_config = dict(grad_clip=None)
# learning policy
lr_config = dict(
    policy='step',
    warmup='linear',
    warmup_iters=500,
    warmup_ratio=0.001,
    # [7] yields higher performance than [6]
    step=[7])
**total_epochs = 8  # actual epoch = 8 * 8 = 64**
log_config = dict(interval=100)

Thanks!

**Comment by hiyyg** [comment]
Because the repeat times is 8.
